remote/ethbackend.proto

```diff
@@ -83,8 +82,8 @@ message EngineForkChoiceUpdated {
 }
 
 message EngineForkChoiceUpdatedRequest {
-  EnginePreparePayload headBlockHash = 1;
-  EngineForkChoiceUpdated safeBlockHash = 2;
+  EnginePreparePayload prepare = 1;
```

Comment:
Perhaps it's just nitpicking, but I would rather have
```
EngineForkChoiceUpdated forkchoice = 1;
EnginePreparePayload prepare = 2;
```
to match the param order in the [Engine API](https://github.com/ethereum/execution-apis/blob/v1.0.0-alpha.5/src/engine/specification.md#request-1).